.github/workflows/php.yml

```diff
@@ -25,13 +33,13 @@ jobs:
       uses: actions/cache@v2
       with:
         path: vendor
-        key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+        key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
         restore-keys: |
-          ${{ runner.os }}-php-
+          ${{ runner.os }}-composer-
 
     - name: Install dependencies
       if: steps.composer-cache.outputs.cache-hit != 'true'
-      run: composer install --prefer-source --no-interaction
+      run: composer install --prefer-dist --no-interaction
```

Comment:
this is the default behavior. I think we can omit it

Reply:
Could you explain, what is the default behavior?

Reply:
Btw, maybe it's better to switch to testing on the lowest and highest dependencies? What do you think?

Reply:
Switched to https://github.com/marketplace/actions/install-composer-dependencies, it supports also running on the lowest & highest dependencies